wiki/misc/qmjfmj.md

```diff
@@ -0,0 +1,127 @@
+---
+title: Differences between the QMJ and the FMJ
```

Comment:
I'm not sure I like this article in its current form. maybe something listing new features in the QMJ vs FMJ? I'm not there's much utility to this, as I know if I was porting my mod I'd rather just look at the QTM QMJ and port based on that.

Reply:
ehhh disagree, I tend to look at docs first, and only fall back to looking at the code if the docs aren't sufficient